TODO(@thunder-coding): Remove this patch when NDK update fixes the issue.
--- ./deps/v8/src/handles/handles.h.orig	2024-02-18 16:58:52.371208097 +0530
+++ ./deps/v8/src/handles/handles.h	2024-02-18 16:59:18.375124836 +0530
@@ -129,7 +129,7 @@
     } else {
       // `static_assert(false)` in this else clause was an unconditional error
       // before CWG2518. See https://reviews.llvm.org/D144285
-#if defined(__clang__) && __clang_major__ >= 17
+#if defined(__clang__) && __clang_major__ >= 17 && !defined(__ANDROID__)
       // For non-HeapObjects, there's no on-heap object to dereference, so
       // disallow using operator->.
       //
